genre.rb

```diff
@@ -0,0 +1,11 @@
+require './item'
+
+class Genre
+  attr_accessor :name, :items
+
+  def initialize(name, _items)
+    @id = Random.rand(1..1000)
+    @name = name
+    @items = []
+  end
+end
```

Comment:
Please add an add_item method to the Genre class as required by the project. The add_item method should,
- take an instance of the Item class as an input.
- add the input item to the collection of items.
- add self as a property of the item object (by using the - correct setter from the item object).

Please check the [requirement](https://github.com/users/Haywayaheadshot/projects/7/views/1?pane=issue&itemId=18112842)